src/Repository/ArticleRepository.php

```diff
@@ -46,7 +41,9 @@ public function findByCategoryPaginated(string $category, int $page, int $perPag
         $qb = $this->createQueryBuilder('a')
             ->select('a', 'm')
             ->leftJoin('a.media', 'm')
+            ->leftJoin('a.category', 'category')
```

Comment:
why left join and not inner?

Reply:
good question , I think it s better, but why no inner join for 

```

            $qb
                ->addSelect('c')
                ->leftJoin('a.category', 'c')
                ->andWhere('c.slug = :category')
                ->setParameter('category', $category)
            ;
```

?

Reply:
I will transform theses things with inner join ^^, it's more logical